Source/WebCore/platform/graphics/GraphicsLayer.h

```diff
@@ -849,6 +858,10 @@ class GraphicsLayer : public RefCounted<GraphicsLayer> {
     FloatRoundedRect m_backdropFiltersRect;
     std::optional<FloatRect> m_animationExtent;
 
+#if HAVE(CORE_MATERIAL)
+    AppleVisualEffect m_appleVisualEffect { AppleVisualEffect::None };
```

Comment:
Could you find an 8-bit member to group this with to optimize padding?

Reply:
Moved in between two members where there was 1 byte of padding.